### PR TITLE
Add automation for AWS -> downstream consumers

### LINKED ISSUE #30: Add automation for AWS -> downstream consumers
Create automation to create issues in downstream consumers of pulumi-aws whenever a release occurs:

1. pulumi-awsx
2. pulumi-eks
3. pulumi-aws-apigateway

Avoid the use of Zapier if possible because it requires manual config outside of IaC.

### PR DESCRIPTION
**Pull request by jkodroff**
Fixes #30 

Example of a workflow run that triggers the Lambda: https://github.com/pulumi/fake-aws/actions/runs/2826546282
Example of a generated issue: https://github.com/pulumi/fake-awsx/issues/3

### CODE REVIEW

**Comment by jkodroff** [comment]
The Lambda is showing an unexpected diff. Tested deploying on my machine to the dev account, and it looks like the Lambda is ok.

File: lambda/internal-release-handler/main.go [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+func getDownstreamRepos(upstreamRepoName string) []string {
+	switch upstreamRepoName {
+	case "fake-aws":
+		return []string{"fake-awsx"}
+	case "pulumi-aws":
+		return []string{"pulumi-awsx", "pulumi-eks", "pulumi-aws-apigateway"}
+	default:
+		return []string{}
```
Comment (jkisk) [inline lambda/internal-release-handler/main.go:199]: would it make sense to have this return an error also so you could return an helpful error in the default case, or perhaps log a message?

File: pulumi/go.mod [hunk 2]
```diff
@@ -1,9 +1,57 @@
 module github-issue-automation
 
-go 1.16
+go 1.19
 
 require (
 	github.com/pulumi/pulumi-aws/sdk/v4 v4.31.0
 	github.com/pulumi/pulumi/sdk/v3 v3.19.0
```
Comment (jkisk) [inline pulumi/go.mod:7]: are we able to use a newer pulumi sdk now that we are upgrading go version?

**Review by jkisk — APPROVED** [review]
Looks great!  A couple suggestions inline that you can incorporate if you find useful.

File: lambda/internal-release-handler/main.go (hunk 1 above)
Comment (jkodroff) [reply lambda/internal-release-handler/main.go:199]: I agree - that's why the caller logs a message!

https://github.com/pulumi/github-issue-automation/pull/31/files#diff-bbbe637f3c28eb20497cc5df51b93b397e194707854eb656d06c1a01432a325aR52

File: pulumi/go.mod (hunk 2 above)
Comment (jkodroff) [reply pulumi/go.mod:7]: Yes - added https://github.com/pulumi/github-issue-automation/issues/33

**Pull request merged**